src/abstract/CSBondCurve.sol

```diff
@@ -157,7 +157,6 @@ abstract contract CSBondCurve is ICSBondCurve, Initializable {
         uint256[] calldata defaultBondCurvePoints
     ) internal onlyInitializing {
         uint256 addedId = _addBondCurve(defaultBondCurvePoints);
-        // TODO: Figure out how to test it
```

Comment:
what we decided?

Reply:
It is already tested. Just forgot to remove this todo.